.gitignore

```diff
@@ -75,7 +75,6 @@ Session.vim
 # auto-generated tag files
 tags
 ### VisualStudioCode ###
-.vscode/*
```

Comment:
To avoid committing other changes set by VS Code (if developing locally, for example), this should maybe be
```gitignore
.vscode/*
!.vscode/extensions.json
!.vscode/launch.json
```

Reply:
Right. Updated, thanks.